package.json

```diff
@@ -73,7 +73,7 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@doc-tools/transform": "^2.9.0",
     "@gravity-ui/eslint-config": "^2.0.0",
-    "@gravity-ui/page-constructor": "^2.13.0",
+    "@gravity-ui/page-constructor": "^3.0.0-alpha.3",
```

Comment:
Are we going to release it with alpha?

Reply:
no, after p-c release we create own release